pkg/systemd/overview-cards/configurationCard.jsx

```diff
@@ -30,6 +30,7 @@ import { EmptyStatePanel } from "cockpit-components-empty-state.jsx";
 import { ServerTimeConfig } from 'serverTime.js';
 import { RealmdClient, RealmButton } from "./realmd.jsx";
 import { TunedPerformanceProfile } from '../../tuned/dialog.jsx';
+import { CryptoPolicyRow } from './cryptoPolicies.jsx';
```

Comment:
Please fix the syntax in the commit message: `Close #8519` → `Fixes #8519`. (Traditionally we used "Close*s*" for closing PRs, and "Fixes" for fixing bugs)